pkg/playbook/app/vcertutil/vcertutil.go

```diff
@@ -91,6 +95,28 @@ func buildClient(config domain.Config, zone string) (endpoint.Connector, error)
 		LogVerbose:      false,
 	}
 
+	vcertConfig.Client = &http.Client{
+		Timeout: time.Duration(DefaultTimeout) * time.Second,
+	}
+	if timeout > 0 {
+		vcertConfig.Client.Timeout = time.Duration(timeout) * time.Second
+	}
+	var connectionTrustBundle *x509.CertPool
+
+	if vcertConfig.ConnectionTrust != "" {
+		zap.L().Debug("Using trust bundle in custom http client")
```

Comment:
Don't just invoke zap, pass the actual logger used by the program. This way zap will create a default logger without the customizations from the plyabook